.swagger-codegen-ignore

```diff
@@ -23,7 +23,11 @@
 #!docs/README.md
 README.md
 Gemfile
+app/models/*.rb
+app/controllers/*.rb
```

Comment:
@syncrou should this also be specific files

Reply:
Same